lnwire/custom.go

```diff
@@ -2,13 +2,62 @@ package lnwire
 
 import (
 	"bytes"
-	"errors"
+	"fmt"
 	"io"
 )
 
-// CustomTypeStart is the start of the custom type range for peer messages as
-// defined in BOLT 01.
-var CustomTypeStart MessageType = 32768
+var (
+	// CustomTypeStart is the start of the custom type range for peer
+	// messages as defined in BOLT 01.
+	CustomTypeStart MessageType = 32768
```

Comment:
Nit: looks like `CustomTypeStart` should be const.

Reply:
Pre-existing (this is a var in master) - will pull it out into a const in refactor commit.

Reply:
Added in c89dce6.